projects/pycryptodome/block_common.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/block_common.c b/src/block_common.c
+index 169200a6..c6da1f85 100644
+--- a/src/block_common.c
++++ b/src/block_common.c
+@@ -106,7 +106,7 @@ EXPORT_SYM int CIPHER_START_OPERATION(const uint8_t key[], size_t key_len, CIPHE
+     if ((key == NULL) || (pResult == NULL))
+         return ERR_NULL;
+
+-    *pResult = calloc(1, sizeof(CIPHER_STATE_TYPE));
```

Comment:
why is this change needed?

Reply:
My fuzzers are written in C++, but PyCryptodome is written in C. C supports implicit casting from the result of `calloc()`, but C++ doesn't, which is why I was getting build errors. Normally, I'd write my fuzzers in C to compensate, but I wanted to take advantage of the `FuzzedDataProvider` class.